pkg/agent/route/route_linux.go

```diff
@@ -409,6 +410,8 @@ func (c *Client) syncIPSet() error {
 		})
 	}
 
+	// FlexibleIPAM is available in noEncap mode. There is a validation in Antrea configuration about this traffic mode
+	// when FlexibleIPAM is enabled.
```

Comment:
Can we use AntreaIPAM here?  If you mean AntreaIPAM feature gate.
I actually didn't see a feature gate named `FlexibleIPAM`. Sometime it's confusing to see both are used.

Reply:
Done